packages/plugin-team/test/integration/spec/actions.js

```diff
@@ -0,0 +1,525 @@
+          kirk.spark.team.archive(conversation),
+          kirk.spark.team.archive(team)
+        ]);
+      })
+      .then(() => kirk.spark.team.get(team, {includeTeamConversations: true}))
+      .then((t) => {
+        team = t;
+        conversation = find(team.conversations.items, {id: conversation.id});
+      }));
+
+      it(`unarchives a team conversation`, () => {
+        assert.include(conversation.tags, `ARCHIVED`);
+        assert.include(conversation.tags, `HIDDEN`);
+
+        return kirk.spark.team.unarchive(conversation)
+          .then((activity) => {
+            assert.isActivity(activity);
+            assert.equal(activity.verb, `unarchive`);
+            assert.equal(activity.target.id, conversation.id);
+            assert.equal(activity.object.id, conversation.id);
+
+            return kirk.spark.team.get(team, {includeTeamConversations: true});
+          })
+          .then((t) => {
+            conversation = find(t.conversations.items, {id: conversation.id});
+            assert.isDefined(conversation);
+            assert.notInclude(conversation.tags, `ARCHIVED`);
+            assert.notInclude(conversation.tags, `HIDDEN`);
+          });
+      });
+
+      it(`unarchives a team`, () => {
+        assert.isTrue(team.archived);
+
+        return kirk.spark.team.unarchive(team)
+          .then((activity) => {
+            assert.isActivity(activity);
+            assert.equal(activity.verb, `unarchive`);
+            assert.equal(activity.target.id, team.id);
+            assert.equal(activity.object.id, team.id);
+
+            return kirk.spark.team.get(team, {includeTeamConversations: true});
+          })
+          .then((t) => {
+            assert.isFalse(t.archived);
+
+            const generalConversation = find(t.conversations.items, {id: team.generalConversationUuid});
+            assert.isDefined(generalConversation);
+            assert.notInclude(generalConversation.tags, `ARCHIVED`);
+            assert.notInclude(generalConversation.tags, `HIDDEN`);
+          });
+      });
+    });
+
+    describe(`#update()`, () => {
+
+      let team;
+
+      before(() => kirk.spark.team.create({
+        displayName: `team-${uuid.v4()}`,
+        participants: [
+          kirk
+        ]
+      })
+      .then((t) => {
+        team = t;
+      }));
+
+      it(`updates a team displayName`, () => {
+        const obj = {
+          displayName: `updated-team-title-${uuid.v4()}`,
+          objectType: `team`
+        };
+
+        return kirk.spark.team.update(team, obj)
+          .then((activity) => {
+            assert.isActivity(activity);
+            assert.equal(activity.verb, `update`);
+            assert.equal(activity.target.id, team.id);
+          })
+          .then(() => kirk.spark.team.get(team))
+          .then((t) => {
+            assert.equal(t.displayName, obj.displayName);
+          });
+      });
+
+      it(`updates a team summary`, () => {
+        const obj = {
```

Comment:
The object has different properties for each test, so I think I'll keep these cases this way